src/libcrun/cgroup.c

```diff
@@ -736,7 +738,71 @@ systemd_job_removed (sd_bus_message *m, void *userdata, sd_bus_error *error arg_
     return -1;
 
   if (strcmp (p->path, path) == 0)
-    *p->terminated = 1;
+    {
+      p->terminated = 1;
+      p->unit = xstrdup(unit);
+      p->result = xstrdup(result);
+    }
+  return 0;
+}
+
+static int
+systemd_check_job_status_setup (sd_bus *bus,
+                                void **data,
+                                libcrun_error_t *err)
+{
+  int ret;
+
+  *data = xmalloc(sizeof(struct systemd_job_removed_s));
```

Comment:
Forgot the ` memset` :(

Reply:
coding style (missing space after `(')

Reply:
don't worry about the coding style, I will re-format and add a check for it with `indent` after the release :-)